app/assets/javascripts/orders.coffee

```diff
@@ -40,15 +40,18 @@ findLastCategory = ->
   orders = $('.well').find '.order'
   $(orders[orders.length-1]).find '#category'
 
-$(document).on "change", ".form-control", (e) ->
+$(document).on "change", ".orders-list .form-control", (e) ->
```

Comment:
Please see #182, I think a general `auto-submit` removes the need for the `orders-list` scope, yet is also available for similar purposes rather than requiring the same javascript for each page :-)